src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a + '' < b + '') {
+        return -1;
+      }
+
+      return 1;
```

Comment:
The default compare function does not correctly handle the case when `a` and `b` are equal. It should return 0 when the compared elements are equal to prevent unnecessary swaps and to maintain stability.